src/instruments/src/PFD/shared/PFDSimvarPublisher.tsx

```diff
@@ -55,7 +60,6 @@ export interface PFDSimvars {
     markerBeacon: number;
     isAltManaged: boolean;
     targetSpeedManaged: number;
-    mach: number;
```

Comment:
Probably other Arinc429 words could be removed as well or? But not needed right now, better suited in another PR I think

Reply:
Roll, pitch etc I mean :D

Reply:
Yeah, later PR I think